src/Server/modules/WorldPlaceManager.js

```diff
@@ -0,0 +1,245 @@
+const dataClassesModule = require('../modules/DataClasses.js');
+const readCSVModule = require('../modules/ReadCSV.js');
+const writeCSVModule = require('../modules/WriteCSV.js');
+
+// Array of WorldPlaces read from CSV
+var allWorldPlace = [];
+
+// Populates allWorldPlace with data read from CSV
+function InitializeAllWorldPlace() {
+    allWorldPlace = readCSVModule.LoadWorldData();
+}
+
+function AddallWorldPlace(province, country, date, tod) {
```

Comment:
When we add new data, we also need to modify currentNumConfirmed/Deaths/Recovered in allWorldPlace[i].